test/RibbonThetaVault.ts

```diff
@@ -135,15 +135,17 @@ describe("RibbonThetaVault", () => {
     strikeAsset: USDC_ADDRESS[chainId],
     collateralAsset: USDC_ADDRESS[chainId],
     chainlinkPricer: CHAINLINK_WETH_PRICER_NEW[chainId],
-    deltaFirstOption: BigNumber.from("1000"),
-    deltaSecondOption: BigNumber.from("1000"),
-    deltaStep: BigNumber.from("100"),
-    depositAmount: BigNumber.from("100000000000"),
+    deltaFirstOption: BigNumber.from("100"),
+    deltaSecondOption: BigNumber.from("100"),
```

Comment:
you need to account for these changes in precision in strike selection.

100 right now is 0.01 delta
1000 is 0.1 delta
10000 is 1 delta

https://github.com/ribbon-finance/ribbon-v2/blob/d3b423508a42cb8f98b69be0436f608baea98e7b/contracts/utils/StrikeSelection.sol#L24

Reply:
Fixed